templates/partials/base.html.twig

```diff
@@ -37,40 +37,50 @@
 <body id="top" class="{% block body_classes %}{{ body_classes }}{% endblock %}">
     <div id="page-wrapper">
     {% block header %}
-        <section id="header" class="section">
-            <section class="container {{ grid_size }}">
-                <nav class="navbar">
-                    <section class="navbar-section logo">
-                        <a href="{{ home_url }}" class="navbar-brand mr-10">{% include('@images/grav-logo.svg') %}</a>
-                    </section>
-                    <section class="navbar-section">
-
-                        <nav class="dropmenu animated">
+        <div id="header" class="section">
+        <div class="container {{ grid_size }}">
+            <div class="navbar">
+             <header class="navbar-section logo" role="banner">
+               <h1 class="navbar-section logo"><a href="{{ home_url }}" class="navbar-brand mr-10">{% include('@images/grav-logo.svg') %}</a></h1>
```

Comment:
I do not recommend to use h1 here, as this completely breaks search engine optimization (seo).